From b9ccd67b35f64ead4d69723e4b13e71886384bec Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 26 Feb 2019 16:56:18 +0100
Subject: [PATCH] spi-grx500: Add condition around variable timeout

The variable is only used inside of this condition, this fixes a compile warning.
---
 drivers/spi/spi-grx500.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/spi/spi-grx500.c b/drivers/spi/spi-grx500.c
index a999b77d09b3..e9680780075f 100644
--- a/drivers/spi/spi-grx500.c
+++ b/drivers/spi/spi-grx500.c
@@ -274,10 +274,8 @@ static void ltq_spi_reset_fifos(struct ltq_spi *hw)
 static inline int ltq_spi_wait_ready(struct ltq_spi *hw)
 {
 	u32 stat;
-	unsigned long timeout;
-
 	#ifndef NO_TIMEOUT
-	timeout = jiffies + msecs_to_jiffies(200);
+	unsigned long timeout = jiffies + msecs_to_jiffies(200);
 	#endif /* NO_TIMEOUT */
 
 	do {
